#! /bin/sh
patch -p0 -f $* < $0
exit $?

Correct default if there is no suffix.

--- src/gpgex.cc	2009-07-17 18:19:12.000000000 +0200
+++ src/gpgex.cc	2010-02-10 11:29:17.000000000 +0100
@@ -213,6 +213,8 @@
 			  if (gpg == false)
 			    this->all_files_gpg = FALSE;
 			}
+		      else
+			this->all_files_gpg = FALSE;
 
 		      this->filenames.push_back (filename);
 		    }
